### PR TITLE
git: add subversion variant

### PR DESCRIPTION
**Pull request by junghans**

### CODE REVIEW

**Comment by junghans** [comment]
@scheibelp @adamjstewart

File: var/spack/repos/builtin/packages/git/package.py [hunk 1]
```diff
@@ -145,6 +145,8 @@ class Git(AutotoolsPackage):
             placement='git-manpages',
             when='@{0}'.format(release['version']))
 
+    variant('subversion', default=False, description='Build svn-fe for git svn')
```
Comment (scheibelp) [inline var/spack/repos/builtin/packages/git/package.py]: It looks like `git svn` is already supported without this variant, so IMO the name of this variant could be more specific: `include_svnfe`?

**Comment by adamjstewart** [comment]
Can you address @scheibelp's comment?

**Comment by junghans** [comment]
Meh, on a second thought it seems not very useful anymore.

**Pull request closed without merging**